index.html

```diff
@@ -0,0 +1,424 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/38b6d4ab77.js"
+      crossorigin="anonymous"
+    ></script> 
+    <link rel="stylesheet" href="./css/style.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Meera+Inimai&family=Poppins&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Zunair Desktop Portfolio Website</title>
+  </head>
+
+  <body>
+    <div class="container">
+
+      <header class="header-section">
+
+        <nav class="nav-bar">
+          <div class="left-navbar">
+            <a href="#" class="logo">Muhammad Zunair khan</a>
+            <div class="wrap">
+              <div class="menu"></div>
+              <div class="menu"></div>
+              <div class="menu"></div>
+            </div>
+          </div>
+  
+          <div class="right-navbar">
+            <ul>
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+              <li><a href="#"><i class="fa fa-envelope-o"></i></a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="headline-section">
+        <div class="headline-container">
+
+          <div class="heads-container">
+            <div class="heading-1">
+              <h1>Hey There. I'm Zunair</h1>
```

Comment:
- [ ] You have done a great job by using a background color in styling 👏 . However, you are missing out on the background image. Kindly ensure to add the background image to look exactly like the Figma design 👌 

Your design 
![back](https://user-images.githubusercontent.com/99394374/213868181-c7cfc387-25e8-4f23-8cbe-c1da67347875.PNG)

Reply:
I hope that you might not see it well background is there. Attached screenshot below.

![image](https://user-images.githubusercontent.com/116083031/213870467-6f201589-95fb-4fff-befd-5e3669d32c1e.png)